ua/org.eclipse.pde.ua.ui/src/org/eclipse/pde/internal/ua/ui/editor/cheatsheet/comp/CompCSMasterTreeSection.java

```diff
@@ -354,16 +353,9 @@ private void handlePreviewAction() {
 		IEditorInput input = getPage().getEditorInput();
 		URL url = null;
 		try {
-			if (input instanceof IFileEditorInput) {
-				IFileEditorInput fileInput = (IFileEditorInput) input;
+			if (input instanceof IFileEditorInput fileInput) {
 				url = fileInput.getFile().getLocationURI().toURL();
-			} else if (input instanceof IStorageEditorInput) {
-				// Note: This URL does not exist on the local file system
-				// As a result any tasks this composite cheat sheet has that
-				// specify a pathes to simple cheat sheets will not resolve
-				// Cheat sheet view will log an error loading simple cheat
-				// sheets
-				IStorageEditorInput storageInput = (IStorageEditorInput) input;
+			} else if (input instanceof IStorageEditorInput storageInput) {
```

Comment:
Please keep the comment .

Reply:
Is this a bug in the cleanup command?

Reply:
> Is this a bug in the cleanup command?

Hi @vik-chand  ,
I did not remove the comment manually. It was done as part of the code cleanup.